src/dlomix/data/ion_mobility.py

```diff
@@ -0,0 +1,152 @@
+from typing import Callable, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+from numpy.typing import NDArray
+from scipy.optimize import curve_fit
```

Comment:
I will move this import to the function just to avoid any failures after installing and importing the package until we decide if we add `scipy` as an install_extras or similar, what do you think?